src/assets/objects/header.objects.ts

```diff
@@ -0,0 +1,37 @@
+export interface menuItemsInterface {
+	itemName: string;
+	routerLink: string;
+	activeOptions?: Object;
+}
+
+export interface Object {
+	exact: boolean;
+}
+
+export const menuItems: menuItemsInterface[] = [
```

Comment:
interfaces shouldn't be in assets, move it in models folder

Reply:
done